backend/src/db/db.client.ts

```diff
@@ -0,0 +1,32 @@
+require("dotenv").config();
+const { MongoClient } = require("mongodb");
+import logger from '../util/winston.createLogger'
+
+
+const { dbURI } =  process.env as {
+    [key: string]: string;
+  };
+
+
+
+interface dbOptionsType {
+    useUnifiedTopology: boolean;
+    useNewUrlParser: boolean;
+}
+
+const dbOptions: dbOptionsType = {
+  useUnifiedTopology: true,
+  useNewUrlParser: true,
+};
+
+const client = new MongoClient(dbURI, dbOptions);
+
+module.exports = async (): Promise<any> => {
```

Comment:
jeżeli funkcja nic nie zwraca to dajemy void

Reply:
Czyli w innym przypadku muszę otypowac, jeżeli funkcja coś zwraca to muszę otypowac ta zwrucona wartość

Reply:
no akurat każda funkcja asynchroniczna zwraca promise w tym wypadku promise który po resolvie da wartość void, jeżeli resolve promise zwróci inna wartość to wtedy musisz ja zwrzeć pomiędzy klamrami np Promise<string>. w wypadku funckji synchronicznej zwrotka wygląda tak (): number